internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -296,17 +286,6 @@ func (r *ClusterDeploymentsReconciler) createNewCluster(
 		UserManagedNetworking: swag.Bool(isUserManagedNetwork(clusterDeployment)),
 	}
 
-	installEnv, err := getInstallEnvByClusterDeployment(ctx, r.Client, clusterDeployment)
```

Comment:
How will newly created `clusterDeployments` get those params? in the next iteration of InstallEnv reconcile?

Reply:
You need it only when creating an ISO so new cluster don't really have a use for those fields.